controllers/v1/observationsController.js

```diff
@@ -730,13 +757,20 @@ module.exports = class Observations extends Abstract {
                 let solutionDocumentFieldList = await observationsHelper.solutionDocumentFieldListInResponse()
 
                 response.result.solution = await _.pick(solutionDocument, solutionDocumentFieldList);
+                response.result.program = programDocument[0];
 
                 let submissionDocument = {
                     entityId: entityDocument._id,
                     entityExternalId: (entityDocument.metaInformation.externalId) ? entityDocument.metaInformation.externalId : "",
                     entityInformation: entityDocument.metaInformation,
                     solutionId: solutionDocument._id,
                     solutionExternalId: solutionDocument.externalId,
+                    programId : programDocument[0]._id,
+                    programExternalId : programDocument[0].externalId,
+                    isAPrivateProgram : programDocument[0].isAPrivateProgram ? programDocument[0].isAPrivateProgram : false,
```

Comment:
@aman-tunerlabs - Do we need to check this condition? There will always be some value right, either true or false, we should just pass that i think